pages/tutorials/okta.md.erb

```diff
@@ -1,23 +1,14 @@
-# Single Sign-on with Okta
+# Single Sign-On with Okta
 
-Okta can be used can be used to authorize access to your Buildkite organization.
-
-## Setting up SSO
-
-Setting up SSO with Okta requires organization administrator privileges in both Okta and Buildkite.
-
-Self-service SSO provisioning in Buildkite is available via our GraphQL explorer. For further information, see the [GraphQL SSO Setup Guide](/docs/tutorials/graphql-sso). 
-
-### Setup in your Okta organization
-
-The instructions below guide you through adding the Buildkite application to your Okta account. The Buildkite team will enable SSO for your organization, with the details you provide from the end of the setup process.
+Okta can be used as an SSO provider for your Buildkite organization. To complete this tutorial, you will need admin privileges for both Okta and Buildkite.
 
 <div class="Docs__note">
-<p class="Docs__note__heading">Before starting your Okta setup</p>
-<p>Email the Buildkite team at support@buildkite.com to request your unique SAML service URL, you will need this when setting up your Okta application.</p>
```

Comment:
I forgot to ask, is this still needed? I didn’t see it mentioned in the Okta setup instructions before? If it is, maybe we can add it back as a troubleshooting note after this, and mention it in the instructions?

Reply:
I specifically wanted to call it out at the start of the guide, not as part of the steps. Otherwise it's a bit like "step 1, email us and wait". At least when it's outside the instructions as a prerequisite people are able to continue with their setup a bit more speedily once they start? If that makes sense?

Reply:
And yes it's still needed 😄

Reply:
That’s cool. I was just a bit confused, because if you follow the instructions to email for the URL, it then doesn’t appear in the steps below. Where do you copy and paste in the URL?

Reply:
I thought it was part of the instructions on the Okta side, but I just checked and it’s totally not! So nevermind about adding it back 👍🏻 

I also realised that we can just tell people about the metadata URL straight off the bat and not worry about getting them to follow the okta instructions at all - okta provides a link to their metadata on their sso setup page already 👌🏻